packages/flutter/test/cupertino/route_test.dart

```diff
@@ -2431,6 +2432,53 @@ void main() {
       expect(tester.getBottomRight(find.byType(Placeholder)).dx, 390.0);
     });
   });
+
+  testWidgets(
+  // TODO(polina-c): remove when fixed https://github.com/flutter/flutter/issues/145600 [leak-tracking-opt-in]
+  experimentalLeakTesting: LeakTesting.settings.withTracked(classes: <String>['CurvedAnimation']),
+  'Fullscreen route does not leak CurveAnimation', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Builder(
+          builder: (BuildContext context) {
```

Comment:
I think you can remove this `Builder` widget, right?

Reply:
Actually we need this for accessing `context` for `Navigator.push`